### ISSUE TITLE
Why are not all `.cfg` files structured the same way?

### ISSUE THREAD

**Issue by youralien** [santosfamilyfoundation/TrafficCloud#14]
some cfg files have section headers, like {project_name}.cfg
```
[homography]
unitpixelratio = 0.055
```

while others, like run_tracking.cfg do not.  The section headers are really nice, because we can use libraries like ConfigParser to better parse, rather than handling it like so

```

    f = open(path1, 'r')
    lines = f.readlines()
    f.close()
    with open(path1, "w") as wf:
        for line in lines:
            line_param = line.split('=')[0].strip()
            if "frame1" == line_param:  # Replace parameter "frame1"
                wf.write("frame1 = 0\n")
            elif "nframes" == line_param:  # Remove parameter "nframes"
                wf.write("nframes = 0\n")
            elif "database-filename" == line_param:
                wf.write("database-filename = results.sqlite\n")
            else:
                wf.write(line)
```

**Comment by youralien** [comment]
A new PR is up that implements reading and updating functionality for cfg files that cannot be used with ConfigParser.  IMO, it's a good enough interface that we can continue on WITHOUT changing the cfg file format that TrafficIntelligence sets out.  This is probably a better thing for not changing stuff up on maintainers like Nicolas.  https://github.com/santosfamilyfoundation/TrafficCloud/pull/15

**Comment by runnersaw** [comment]
Can this be closed now that #15 has been merged, or is this a more open question than "How can we work with each type of file?"?

**Comment by youralien** [comment]
Yup since #15 has been merged, I think we can close this and say the issue has been solved from a UI perspective (good programatic interfaces for different types of config files) without having to worry about a standard on format.
